client/src/App.js

```diff
@@ -79,6 +78,11 @@ class App extends Component {
           render={() =>
             <CreateJob/>}
         />
+        <Route
```

Comment:
You can probably throw these Routes as 1 line It's hard to read when we can just see the [Route, path, component] all in one line and you can see your routes listed out